README.md

```diff
@@ -1,125 +1,82 @@
 
-### Using Openshift CLI
 
-- Make sure that `oc` is configured to talk to the cluster
-- Deploy the operator by running the following
-  ```
-  oc apply -f https://raw.githubusercontent.com/openshift/sandboxed-containers-operator/master/config/samples/deploy.yaml 
-  ```
+### Features & benefits of sandboxed containers
 
-- Check if operator pods are up (usually takes few minutes)
-  ```
-  oc get pods -n openshift-sandboxed-containers-operator
-  ```
-  Sample output
-  ```
-  NAME                                 READY   STATUS    RESTARTS   AGE
-  controller-manager-64888847f-j6256   2/2     Running   0          35s
-  ```
+- **Isolated Developer Environments & Privileges Scoping** 
+As a developer working on debugging an application using state-of-the-art tooling you might need elevated privileges such as CAP_ADMIN or CAP_BPF. With OpenShift sandboxed containers, any impact will be limited to a separate dedicated kernel.
 
-### Using Openshift Web Console
+- **Legacy Containerized Workload Isolation** 
+You are mid-way in converting a containerized monolith into cloud-native microservices. However, the monolith still runs on your cluster unpatched and unmaintained. OpenShift sandboxed containers helps isolate it in its own kernel to reduce risk.
 
-- Switch to `Administrator` perspective and navigate to `Admininstration -> Cluster Settings`.
-- In the `Cluster Settings` page, switch to `Configuration` tab and search for `OperatorHub`
-- Click `OperatorHub` and navigate to `Sources` tab.
-- Click `Create CatalogSource` and use the following `Image`
-  ```
-  quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator-catalog:v1.2.0
-  ```
+- **Safe Multi-tenancy & Resource Sharing (CI/CD Jobs, CNFs, ..)** 
+If you are providing a service to multiple tenants, it could mean that the service workloads are sharing the same resources (e.g., worker node). By deploying in a dedicated kernel, the impact of these workloads have on one another is greatly reduced.
 
-- Ensure `Cluster-wide CatalogSource` is selected.
+- **Additional Isolation with Native Kubernetes User Experience**
+OpenShift sandboxed containers is used as a compliant OCI runtime. Therefore, many operational patterns used with normal containers are still preserved including but not limited to image scanning, GitOps, Imagestreams, and so on.
 
-- Click `Create` to create the catalog. 
+Please refer to this [blog](https://cloud.redhat.com/blog/the-dawn-of-openshift-sandboxed-containers-overview) for a detailed overview of sandboxed containers use cases and other related details.
 
-The sandboxed container operator will be available under `Operators -> OperatorHub`
+## OpenShift sandboxed containers Operator
 
+The operator manages the lifecycle (`install/configure/update`) of sandboxed containers runtime (`Kata containers`) on OpenShift clusters.
```

Comment:
why is "install/configure/update" codified.

Reply:
Just to show it in a different format (without bold or italics). Do you prefer to change it to regular text ?

Reply:
@zanetworker thanks for your comments. I have addressed the additional link. This one I have kept it as is for now. If you think otherwise, let me know and I'll change it to regular text.